code/scripting/api/libs/graphics.cpp

```diff
@@ -2073,7 +2073,7 @@ ADE_FUNC(createPersistentParticle,
 
 ADE_FUNC(createParticle,
 	l_Graphics,
-	"vector Position, vector Velocity, number Lifetime, number Radius, enumeration Type, [number TracerLength=-1, "
+	"vector Position, vector Velocity, number Lifetime, number Radius, [enumeration Type, number TracerLength=-1, "
```

Comment:
This should say what the default is i.e. `[enumeration Type = PARTICLE_DEBUG, ..`